tools/csv-merger/csv-merger.go

```diff
@@ -369,6 +369,10 @@ func unmarshalCSV(filePath string) *csvv1.ClusterServiceVersion {
 				Name:  "ROOK_OBC_WATCH_OPERATOR_NAMESPACE",
 				Value: "true",
 			},
+			{
+				Name:  "ROOK_OBC_PROVISIONER_NAME_PREFIX",
```

Comment:
@thotz now rook is managing its own CSV and this will be removed from the ocs very soon. pls add this change in the DS rook repo.

Reply:
@iamniting are u referring to https://github.com/red-hat-storage/rook/pull/581/files#diff-33db4b9fd0695d15138ddd1de5fe3047aafd2d108e9b0c101024278370ab7c43 ??
It will be easier that change in @subhamkrai PR right?

Reply:
Yes

Reply:
@subhamkrai can please include this setting in u PR

Reply:
sure I'll update on my PR

Reply:
added

Reply:
Thanks

Reply:
@thotz Can you drop this change from here?